action/action-dsl/src/test/java/com/powsybl/action/dsl/ActionDslLoaderTest.java

```diff
@@ -151,6 +186,73 @@ private void addPhaseShifter() {
                     .setAlpha(5.0)
                     .setRho(6.0)
                 .endStep()
+                .beginStep()
+                    .setR(1.0)
+                    .setX(2.0)
+                    .setG(3.0)
+                    .setB(4.0)
+                    .setAlpha(5.0)
+                    .setRho(6.0)
+                .endStep()
+                .beginStep()
+                    .setR(1.0)
+                    .setX(2.0)
+                    .setG(3.0)
+                    .setB(4.0)
+                    .setAlpha(5.0)
+                    .setRho(6.0)
+                .endStep()
                 .add();
     }
+
+    public static class DeltaTapData {
+
+        private int iniTapPosition;
+        private int expectedTapPosition;
+        private int deltaTap;
+        private String testName;
+
+        public DeltaTapData() {
```

Comment:
Remove this and do not `public` if possible.

Reply:
Done !